aten/src/ATen/native/native_functions.yaml

```diff
@@ -1229,6 +1229,12 @@
 
 - func: conv_transpose3d.input(Tensor input, Tensor weight, Tensor? bias=None, int[3] stride=1, int[3] padding=0, int[3] output_padding=0, int groups=1, int[3] dilation=1) -> Tensor
 
+# out-of-place variant of copy_()
+# This doesn't need to be exposed to python, but is needed for functionalization.
```

Comment:
It would be nice if we could add a test for this somehow... Maybe just write a manual test for torch.ops.aten.copy? Should probably test some forward + backward (so we catch things like setting up the gradient formula incorrectly)

Reply:
sounds good - just wrote a gradcheck test for `torch.ops.aten.copy`